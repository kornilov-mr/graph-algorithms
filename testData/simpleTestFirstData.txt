8 15
1 2 5
1 3 2
1 4 5
2 3 6
2 5 4
2 4 3
2 6 5
3 4 3
3 8 8
4 5 3
5 6 2
5 7 3
5 8 3
6 7 2
7 8 3
1