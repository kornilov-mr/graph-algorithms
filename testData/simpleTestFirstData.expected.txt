8
1 0 -1
2 5  1
3 2  1
4 5  1
5 8  4
6 10 2
7 11 5
8 10 3
